### DISCUSSION TITLE
[✨] eslint rule to detect useless track()

### DISCUSSION THREAD

**Discussion by manucorporat** [QwikDev/qwik-evolution#178] [Proposals For Qwik]
### Is your feature request related to a problem?

```tsx
component$(({count}) => {
  useTask(({track}) => {
    track(() => count); // bad, it should be props.count
    track(count); // bad, it should be () => props.count
  });
});
```

### Describe the solution you'd like

A eslint rule that errors when seeing this pattern

### Describe alternatives you've considered

Runtime check when a track() does not create any subscription. Maybe both.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
